src/lib/style/theme.js

```diff
@@ -138,12 +138,12 @@ export const lineColor7 = '#FD8144';
 export const lineColor8 = '#F6B187';
 
 //navbar
-export const navbarHeight = '47px';
+export const navbarHeight = '2rem';
 export const navbarItemWidth = '60px';
 
 //sidebar
-export const sidebarItemHeight = '40px';
-export const sidebarWidth = '40px';
+export const sidebarItemHeight = '1.75rem';
```

Comment:
```suggestion
export const sidebarItemHeight = spacing.sp40;
```
We have spacing defined in the theme.js that map the px to rem, you may want to directly use them..

Reply:
@ChengYanJin Values has been updated

Reply:
I'm updating `spacing` to add new values in order to use them in metalk8s via this [PR](https://github.com/scality/metalk8s/pull/3594), I'll let you know once this PR is ready

Reply:
@ChengYanJin This PR is now ready, I'll update [metalk8s PR](https://github.com/scality/metalk8s/pull/3594) once this one is merged and released